### PR TITLE
Updated Russian Language

### PR DESCRIPTION
**Pull request by Nekiplay**
Removed bad spaces
Удалены лишние пробелы

### CODE REVIEW

File: MinecraftClient/Resources/lang/ru.ini [hunk 1]
```diff
@@ -498,4 +498,4 @@ bot.scriptScheduler.task=triggeronfirstlogin: {0}\n triggeronlogin: {1}\n trigge
 
 # TestBot
 bot.testBot.told=Bot: {0} сказал мне : {1}
-bot.testBot.said=Бот: {0} сказал : {1}
\ No newline at end of file
+bot.testBot.said=Бот: {0} сказал : {1}
```
Comment (ORelio) [inline MinecraftClient/Resources/lang/ru.ini:501]: Should the space here one also be removed?
```suggestion
bot.testBot.said=Бот: {0} сказал: {1}
```

**Review by ORelio — COMMENTED** [review]
Thanks for contributing!

File: MinecraftClient/Resources/lang/ru.ini (hunk 1 above)
Comment (Nekiplay) [reply MinecraftClient/Resources/lang/ru.ini:501]: Yes

Comment (ORelio) [reply MinecraftClient/Resources/lang/ru.ini]: OK! Applied suggestion and merged PR 👍

**Pull request merged**